L1Trigger/L1THGCal/plugins/backend/HGCalStage1TruncationWrapper.cc

```diff
@@ -49,16 +51,18 @@ void HGCalStage1TruncationWrapper::convertCMSSWInputs(const std::vector<edm::Ptr
   fpga_tcs_SA.reserve(fpga_tcs.size());
   unsigned int itc = 0;
   for (auto& tc : fpga_tcs) {
-    fpga_tcs_SA.emplace_back(tc->position().x(),
-                             tc->position().y(),
-                             tc->position().z(),
-                             triggerTools_.zside(tc->detId()),
-                             triggerTools_.layerWithOffset(tc->detId()),
-                             tc->eta(),
-                             tc->phi(),
-                             tc->pt(),
-                             tc->mipPt(),
-                             itc);
+    const GlobalPoint& position = tc->position();
+    double x = position.x();
+    double y = position.y();
+    double z = position.z();
+    unsigned int digi_rOverZ = (std::sqrt(x * x + y * y) / std::abs(z)) * 10000;  // Magic numbers
```

Comment:
Should probably define `10000` as a named constant, in particular since it is used several times

Reply:
done